### DISCUSSION TITLE
Is there YouTube Music integration?

### DISCUSSION THREAD

**Discussion by alok-old** [KRTirtho/spotube#65] [General]
Is the feature yet to be implemented?

**Comment by KRTirtho** [comment]
No, Spotube fetches data from Spotify's servers & uses YouTube API to only get the track binary
Currently there is no plan to implement YouTube Music integration as its not yet available in most of the countries

**Reply by alok-old** [reply]
I prefer using YouTube Music as there are many unofficial versions (like Nightcore version) of songs which I prefer to the original and also it just know to create a perfect playlist for me. Anyway, the android version is much smoother than the Linux version. Thanks, 🙂
